src/Sentry.Unity/ScriptableSentryUnityOptions.cs

```diff
@@ -38,6 +38,10 @@ public static string GetConfigPath(string? notDefaultConfigName = null)
         [field: SerializeField] public string ReleaseOverride { get; set; } = string.Empty;
         [field: SerializeField] public string EnvironmentOverride { get; set; } = string.Empty;
         [field: SerializeField] public bool AttachStacktrace { get; set; }
+        [field: SerializeField] public bool AttachScreenshot { get; set; }
+        [field: SerializeField] public int ScreenshotMaxWidth { get; set; }
+        [field: SerializeField] public int ScreenshotMaxHeight { get; set; }
```

Comment:
I'm not sure about dealing with absolute pixel sizes here. It does not take into account devices with different aspect ratios or switching from portrait to landscape.
Maybe we should start with a ratio instead?

Reply:
We've discussed in the call to align across other SDKs. @bitsandfoxes is going to gather inputs